src/features/zaaksysteem/components/ZaakAlgemeen.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <section>
+    <utrecht-heading :level="2" modelValue>Algemeen</utrecht-heading>
+
+    <div class="content">
+      <ul>
```

Comment:
Might this be suited for a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)?

Reply:
The request to make this page with minimal effort was made, therefore I don't think it's worth to make this change.

(I do think the description list would be a better fit here, yes, but I'd have to make css changes as well to make this happen).